content/en/tracing/trace_collection/dd_libraries/dotnet-framework.md

```diff
@@ -69,11 +70,69 @@ For a complete list of supported libraries and processor architectures (includin
 
 ### Installation
 
-1. [Install the tracer.](#install-the-tracer)
-2. [Enable the tracer for your service.](#enable-the-tracer-for-your-service)
-3. [Configure the Datadog Agent for APM.](#configure-the-datadog-agent-for-apm)
+1. [Configure the Datadog Agent for APM.](#configure-the-datadog-agent-for-apm)
+2. [Install the tracer.](#install-the-tracer)
+3. [Enable the tracer for your service.](#enable-the-tracer-for-your-service)
 4. [View your live data.](#view-your-live-data)
 
+### Configure the Datadog Agent for APM
```

Comment:
I believe we should move this section in its own page as it should be common to all tracers